Edit PR comment in place instead of creating a new comment (#6)
im-open/process-dotnet-test-results: README.md

<kbd><img src="./docs/failed_tests.png"></img></kbd>

## Inputs
| Parameter                     | Is Required | Default                          | Description                                                                                                             |
|-------------------------------|-------------|----------------------------------|-------------------------------------------------------------------------------------------------------------------------|
| `github-token`                | true        | N/A                              | Used for the GitHub Checks API.  Value is generally: secrets.GITHUB_TOKEN.                                              |
| `base-directory`              | false       | `.` Root Directory of repository | The base directory of where to look for `trx` files.                                                                    |
| `create-status-check`         | false       | true                             | Flag indicating whether a status check with code coverage results should be generated.                                  |

> | `create-pr-comment`           | false       | true                             | Flag indicating whether a PR comment with code coverage results should be generated.                                    |

Reviewer: ```suggestion
| `create-pr-comment`           | false       | true                             | Flag indicating whether a PR comment with code coverage results should be generated.  When `true` the default behavior is to always create a new comment.|
```

> | `update-pr-comment-on-change` | false       | false                            | Flag indicating whether the PR comment should be updated upon changes.                                                  |

Reviewer: ```suggestion
| `update-comment-if-one-exists` | false       | false                            | When `update-pr-comment-on-change` is true, this flag determines whether a new comment is created (default) or if the action updates an existing comment if one is found.|
```

Author: Updated documentation and variable name

<kbd><img src="./docs/failed_tests.png"></img></kbd>

> 

Reviewer: I can't make a comment on the correct line, but in the readme example you'll need to update the version of this action to v2.0.0.   When you commit that change please add `+semver:major` to the commit message so the increment-version-on-merge workflow knows how to increment this appropriately.

| `github-token`                 | true        | N/A                              | Used for the GitHub Checks API.  Value is generally: secrets.GITHUB_TOKEN.                                                                                                 |
| `base-directory`               | false       | `.` Root Directory of repository | The base directory of where to look for `trx` files.                                                                                                                       |
| `create-status-check`          | false       | true                             | Flag indicating whether a status check with code coverage results should be generated.                                                                                     |
| `create-pr-comment`            | false       | true                             | Flag indicating whether a PR comment with code coverage results should be generated. When `true` the default behavior is to always create a new comment.                   |

> | `update-comment-if-one-exists` | false       | false                            | When `update-comment-if-one-exists` is true, this flag determines whether a new comment is created (default) or if the action updates an existing comment if one is found. |

Reviewer: Sorry, I gave you a bad suggestion the first time in the review.
```suggestion
| `update-comment-if-one-exists` | false       | false                            | When `create-pr-comment` is true, this flag determines whether a new comment is created (default) or if the action updates an existing comment if one is found. |
```
